main/test/flix/fuzzers/FuzzDuplicateLines.scala

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2024 Magnus Madsen, Herluf Baggesen
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package flix.fuzzers
+
+import ca.uwaterloo.flix.TestUtils
+import ca.uwaterloo.flix.api.Flix
+import org.scalatest.funsuite.AnyFunSuite
+
+import java.nio.file.{Files, Paths}
+import scala.jdk.CollectionConverters._
+
+class FuzzDuplicateLines extends AnyFunSuite with TestUtils {
+
+  /**
+    * Number of lines to duplicate from each file.
```

Comment:
Is that right? The comment seems misleading.

Reply:
I agree, it is now
```scala
  /**
    * Number of variants to make of each file. Each variant has a single line duplicated.
    */
  private val N = 10
```